taskvine/src/worker/vine_worker.c

```diff
@@ -208,6 +212,90 @@ __attribute__((format(printf, 2, 3))) void send_message(struct link *l, const ch
 	va_end(va);
 }
 
+/* Send messages from list of asychronous messages available*/
```

Comment:
Please add here a big long explanation of why asynchronous messages are needed, how they work, and what the constraints are, so that the next person reading this will get what's going on.